services/vehicle/src/main/java/com/wedriveu/services/vehicle/boundary/CommunicationWithVehiclesImpl.java

```diff
@@ -0,0 +1,64 @@
+package com.wedriveu.services.vehicle.boundary;
+
+import com.rabbitmq.client.ConnectionFactory;
+import com.rabbitmq.client.Connection;
+import com.rabbitmq.client.Channel;
+import com.wedriveu.services.shared.utilities.Util;
+import com.rabbitmq.client.Consumer;
+import com.rabbitmq.client.DefaultConsumer;
+import com.rabbitmq.client.AMQP;
+import com.rabbitmq.client.Envelope;
+import com.wedriveu.services.vehicle.callback.RequestCanDoJourneyCallback;
+
+import java.io.IOException;
+
+/**
+ * Created by Stefano Bernagozzi on 17/07/2017.
+ */
+public class CommunicationWithVehiclesImpl implements CommunicationWithVehicles {
+    private ConnectionFactory factory;
+    private Connection connection;
+    private Channel channel;
+
+    public CommunicationWithVehiclesImpl() throws IOException {
+        factory = new ConnectionFactory();
+        factory.setHost(Util.SERVER_HOST);
+        factory.setPassword(Util.SERVER_PASSWORD);
+        connection = factory.newConnection();
+        channel = connection.createChannel();
+    }
+
+    public void requestCanDoJourney(final String licensePlate,
+                                    double kilometersToDO,
```

Comment:
I would call this field 'kilometersToDo' instead of kilometersToDO. 
use ctrl+f6 to change field/method/interface names in one shot